geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -4838,7 +4838,7 @@ public void fetchAllRemoteEntries(
   /**
    * Test Method: Get the number of entries in the local data store.
    */
-  public long getLocalSize() {
+  public long getLocalSizeForTest() {
```

Comment:
We need to rename the method so that we could use its original name for our new functionality. We saw that numerous methods in Geode ended with 'ForTest()' and went with that. Then we figured that the annotation would be superfluous.